pom.xml

```diff
@@ -43,8 +43,28 @@
 
     <repositories>
         <repository>
-            <id>spigot</id>
-            <url>https://hub.spigotmc.org/nexus/content/groups/public/</url>
```

Comment:
Why aren't you pulling from the official Spigot repository? Doesn't this seem dangerous? There could be malicious code within the libraries you're pulling from there.